docs/specifications/tests/Syntax-TP/syntax06.md

```diff
@@ -17,17 +17,20 @@ The domain name to be tested.
 
 ### Ordered description of steps to be taken to execute the test case
 
-1. Retrieve the SOA record from the zone being tested.
-2. Get the RNAME from the SOA record.
-3. Convert the first non-escaped "." to an "@" in the RNAME.
-4. De-escape the RNAME, converting a "\." to ".".
-5. Verify the result from step 3 and 4 according to the Addr-spec
+1. Obtain a set of name server IP addresses using [Method4] and [Method5].
+2. Create a SOA query for the zone.
+3. Send the SOA query over UDP to each unique name server IP address.
+4. Get the RNAME from the SOA record.
+5. Convert the first non-escaped "." to an "@" in the RNAME.
+6. De-escape the RNAME, converting a "\." to ".".
```

Comment:
Another way of writing 5 and 6:

* Create an empty string.
* Add the first label to the string, if any.
* Exit converting to mail address if RNAME consists of zero or one labels.
* Append an "@" to the string.
* For remaining labels in RNAME:
   * Append the label.
   * Append a "." unless the label was appended.